dwarf-fortress.sh

```diff
@@ -41,10 +47,15 @@ while getopts ":hsdR" opt; do
 done
 
 if [ ! -d /app/dfhack/hack ]; then
-    [ "$DFHACK" = "true" ] && ( echo "This version of com.bay12games.DwarfFortress does not come with dfhack support. Sorry."; exit 1; )
+    [ "$TWBT" = "true" ] && ( echo "This version of com.bay12games.DwarfFortress does not come with dfhack support. Sorry."; exit 1; )
     DFHACK="false"
 fi
 
+if [ ! -d /app/twbt/hack ]; then
+    [ "$DFHACK" = "true" ] && ( echo "This version of com.bay12games.DwarfFortress does not come with TWBT support. Sorry."; exit 1; )
```

Comment:
I think you've got the checks the wrong way around here, this one should probably be `$TWBT`, and the one above should be `$DFHACK`

Reply:
Oh, correct. Fixed